database/src/test/scala/za/co/absa/atum/database/runs/UpdateParentIntegrationTests.scala

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.database.runs
+
+import za.co.absa.balta.DBTestSuite
+import za.co.absa.balta.classes.JsonBString
+
+class UpdateParentIntegrationTests extends DBTestSuite {
```

Comment:
Please follow the name of the sql function so it's easier to find these tests.

Reply:
Understood, Will fix.

Reply:
FYI, the name conversion is from camel_case to TitleCase. + IntegrationTests suffix.